pkg/kn/commands/domain/create.go

```diff
@@ -53,9 +54,14 @@ func NewDomainMappingCreateCommand(p *commands.KnParams) *cobra.Command {
 				return err
 			}
 
+			errs := validation.IsDNS1123Subdomain(refFlags.tls)
```

Comment:
I wonder if we shouldn't validate `name` param similarly.

Reply:
Nevermind that's probably covered by webhook once submitted.

Reply:
Yes, it is covered by webhook :smile: